### ISSUE TITLE
Component: [Accessibility - Screen reader], reading is not working as expected in Carousel

### ISSUE THREAD

**Issue by junan-trustarc** [primefaces/primeng#17023]
### Describe the bug

![image](https://github.com/user-attachments/assets/7fad3419-a07c-463e-a302-1b588204b2a5)
![image](https://github.com/user-attachments/assets/dcd15e99-4889-42a8-b43d-ad1dd289c789)
When next page from navigation icon or below page buttons is clicked, then screen reader doesn't read each item.
I assume screen reader should read next paged items, instead of losing to read

### Environment

PrimeNg 17.x but 18 also same.

### Angular version

18

### PrimeNG version

17.18.x

### Build / Runtime

Angular CLI App

### Language

TypeScript

### Node version (for AoT issues node --version)

20.15.x

### Browser(s)

chrome

### Steps to reproduce the behavior

1. Go to https://v17.primeng.org/carousel
2. Open screen reader
3. go to in carousel using screen reader
4. click next page icon or button 2 in bottom navigation of carousel.
5. click next from screen reader

### Expected behavior

When next page from navigation icon or below page buttons is clicked, then screen reader doesn't read each item.
I assume screen reader should read next paged items, instead of losing to read
